app/controllers/calendars_controller.rb

```diff
@@ -1,19 +1,28 @@
 class CalendarsController < ApplicationController
   before_action :authenticate_user!, except: [:index, :show]
-  before_action :set_calendar, only: [:show, :edit, :update, :destroy]
+  before_action :set_calendar, except: [:index]
 
   # GET /calendars
-  # GET /calendars.json
   def index
-    @calendars = Calendar.all
   end
 
   # GET /calendars/1
-  # GET /calendars/1.json
   def show
-    @calendar_events = @calendar.events.flat_map do |e|
-      e.calendar_events(params.fetch(:start_date, Time.zone.now).to_date)
-    end 
+    @inactive_calendars = []
```

Comment:
I think `inactive` isn't the best word to use here. I initially thought it was an array of calendars that were no longer used and should therefore maybe be deleted. How about just `other_calendars`?

There might also be a better way to do this. You basically just don't want the current calendar yo're on, right? How about: `Calendar.where.not(id: @calendar.id)` (that'll also be more performant, which will matter if you have like... 500 calendars)

You could also look into scopes & see if that's a direction you'd like to go in (now or later) https://guides.rubyonrails.org/active_record_querying.html#scopes

Reply:
So much more elegant and clean, I had no idea about this option `Calendar.where.not(...)` Thank you :balloon: